### ISSUE TITLE
Releasing 2.0.0

### ISSUE THREAD

**Issue by laughedelic** [ohnosequences/statika#17]
@eparejatobes @marina-manrique 
We are using `2.0.0-M4` for quite some time already and no major problems related to the abstract part of statika were reported. So I'm going to release it (_without any changes_) as `2.0.0` and then the same with aws-statika.

So if you want any changes in statika before the major release, it's a good moment to stop me and discuss it. Otherwise, I'll release it tomorrow.

**Comment by eparejatobes** [comment]
after #19 and #

**Comment by laughedelic** [comment]
I have released `2.0.0-M5` with these changes:

* statika and aws-statika are now integrated as one project
* restructured project packages, now only two imports needed:

    ```scala
    import ohnosequences.statika._
    import ohnosequences.statika.aws._
    ```
* Added a constructor for building instructions from a `ProcessBuilder` (withouth `workingDir` management)

**Comment by eparejatobes** [comment]
:+1:

**Comment by laughedelic** [comment]
Released [v2.0.0-RC1](https://github.com/ohnosequences/statika/releases/tag/v2.0.0-RC1)

**Comment by laughedelic** [comment]
RC2 doesn't seem to cause any problems. I'm releasing final v2.0.
